File: beginners/easy/006.temperature-measurement/206.saadman-galib.temperatureMeasurement.py
temp = int(input(" Input days temperature: "))
if temp < 0:
    print("Freezing weather.")
elif temp < 10:
    print("Very cold weather.")
elif temp < 20:
    print("Cold weather.")
elif temp < 30:
    print("Normal in temp.")
elif temp < 40:
    print("It's hot.")
else:
    print("It's very hot.")

File: beginners/easy/006.temperature-measurement/206.saadman-galib.temperatureMeasurement.go
package main

import "fmt"

func main(){
	var temp float64
	fmt.Print("Input days temperature: ")
	fmt.Scan(&temp)
	if temp <0 {
		fmt.Println("Freezing Weather.")
	} else if temp < 10{
		fmt.Println("Very cold weather.")
	} else if temp < 20{
		fmt.Println("Cold Weather.")
	} else if temp < 30{
		fmt.Println("Normal in temp.")
	} else if temp < 40{
		fmt.Println("It's Hot.")
	} else{
		fmt.Println("It's very hot.")
	}
}